Docs for Java SDK Check-ins (#7963)
getsentry/sentry-docs: src/platforms/java/common/configuration/integrations/quartz.mdx

---
title: Quartz Integration

> sidebar_order: 7

Reviewer: Just so it's not the same # as the other integration

```suggestion
sidebar_order: 10
```

description: "Learn how to send check-ins for your Quartz jobs."
---

Sentry's [Quartz](http://www.quartz-scheduler.org/) integration is provided through `SenryJobListener` which you have to add to your scheduler instance.

> Our Quartz integration can be configured automatically if you're using `spring-boot-starter-quartz` with either the `sentry-spring-boot-starter` or the `sentry-spring-boot-jakarta-starter` integration. However you still have to specify the monitor slug as shown below.

Reviewer: ```suggestion
The Quartz integration will be configured automatically if you're using `spring-boot-starter-quartz` with either the `sentry-spring-boot-starter` or the `sentry-spring-boot-jakarta-starter` integration. However, you still have to specify the monitor slug as shown below.
```

> Check-ins may also be [sent manually](/platforms/java/crons/) or by using our [`@SentryCheckIn` annotation](/platforms/java/guides/spring-boot/crons/), in case you are using our Spring Boot integration and would like to send check-ins for your `@Scheduled` tasks.

Reviewer: ```suggestion
Check-ins may also be [sent manually](/platforms/java/crons/) or if you're using Sentry's Spring Boot integration you can send send check-ins for your `@Scheduled` tasks by using our [`@SentryCheckIn` annotation](/platforms/java/guides/spring-boot/crons/).
```
